src/pages/home/report/ReportActionCompose.js

```diff
@@ -236,6 +236,14 @@ class ReportActionCompose extends React.Component {
             this.focus(false);
         });
 
+        // This listener is used for focusing the composer again after going back to a report without remounting it.
+        this.unsubscribeNavFocus = this.props.navigation.addListener('focus', () => {
+            if (!this.willBlurTextInputOnTapOutside || this.props.isFocused || this.props.modal.isVisible) {
+                return;
+            }
+            this.focus();
```

Comment:
This is a bit of a random guess, but I'm seeing a new crash associated and I'm curious if: 
- It's possible this change is the cause
- Or maybe this exception simply occurred during testing?

```
Non-fatal Exception: io.invertase.firebase.crashlytics.JavaScriptError: Cannot read property 'focus' of null
       at .anonymous(address at index.android.bundle:1:2694129)
       at .apply((native):0:0)
       at .anonymous(address at index.android.bundle:1:237083)
       at ._callTimer(address at index.android.bundle:1:236365)
       at .callTimers(address at index.android.bundle:1:238167)
       at .apply((native):0:0)
       at .__callFunction(address at index.android.bundle:1:140250)
       at .anonymous(address at index.android.bundle:1:138615)
       at .__guard(address at index.android.bundle:1:139568)
       at .callFunctionReturnFlushedQueue(address at index.android.bundle:1:138573)
```

Reply:
@ginsuma & @narefyev91

Reply:
this is pretty strange - because we have all needed conditions to not get situation when this.textinput will null and we will try to focus

Reply:
Yeah, I could be wrong here. This was just the only loc that contained a `focus()` call since yesterdays build.

Reply:
> How can I reprocedue it? Or source map?

I'm afraid we can't. That's why this is a bit of a random guess. If we don't think this was the cause then no further action is necessary 👍